README.md

```diff
@@ -7,7 +7,7 @@
 [Temporal](https://temporal.io/) is a distributed, scalable, durable, and highly available orchestration engine used to
 execute asynchronous long-running business logic in a scalable and resilient way.
 
-"Temporal Python SDK" is the framework for authoring workflows and activities using the Python programming language.
+_Temporal Python SDK_ is the framework for authoring workflows and activities using the Python programming language.
```

Comment:
Do we have to change this file now? I would expect when real Python docs are present, this file will be completely rewritten to point there. Also, so much of this file will change with in-flight work that it makes little sense to hyper-focus on a README that not only will be mostly deleted when real docs appear, but is also subject to tons of in-flight changes (this repo is under heavy active development).